dashboard/app/views/devise/registrations/edit.html.haml

```diff
@@ -70,6 +71,9 @@
   - if !resource.teacher?
     .field
       = f.label t('signup_form.age'), class: "label-bold"
+      - if studentInLockoutFlow
+        %em= t('user.update_field_parent_permission_required')
+        %br/
```

Comment:
why don't we just disable it here instead of passing the state to the JavaScript to disable it there?